cmd/ferretdb/main.go

```diff
@@ -513,15 +492,13 @@ func run() {
 
 			l := logging.WithName(logger, "dataapi")
 
-			var lis *dataapi.Listener
-
-			lis, err = dataapi.Listen(&dataapi.ListenOpts{
```

Comment:
Here, in an attempt to avoid shadowing, we used shared `err` from multiple goroutines